tools/n64tool.c

```diff
@@ -71,7 +70,7 @@ uint32_t get_file_size(FILE *fp)
 
 int swap_bytes(uint8_t *buffer, int size)
 {
-	if((size & 1) == 0)
+	if(size & 1 == 0)

```

Comment:
paranthesis would be nice to keep if there is not a specific reason?

Reply:
This is also reverting my change, and thus re-introducing a bug (the parentheses change the operator evaluation order).